aries-site/src/tests/navigation/homepage.js

```diff
@@ -27,3 +27,18 @@ test('should navigate to correct path when a tile is choosen via keyboard', asyn
     .expect(getLocation())
     .contains(expectedPath);
 });
+
+/*Added range to this test since now there is some issues with different browsers as to where
+the page actually lands when routed to a deep link
+*/
+
+fixture('Deep linking').page(baseUrl);
+
+test('should navigate to correct hash of page when a deep link is directly routed to', async t => {
+  const url = `${baseUrl}/foundation/color#background-colors`
+  const pageSection = Selector('#background-colors');
+  await t
+    .navigateTo(url)
+    .expect(pageSection.getBoundingClientRectProperty("top"))
+    .within(-150, 150);
```

Comment:
How those numbers had been chosen?

Reply:
I got these numbers from  doing `document.getElementById('background-colors').getBoundingClientRect()` again this should be equal to 0 but since we have that deep linking bug I did a wider range so that way the tests don't always fail until we get that fixed